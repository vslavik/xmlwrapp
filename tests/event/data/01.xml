<root>
<!-- just under root -->
    <one>
	<one_dot_one this="that"/>
    </one>

    <two>
	<two_dot_one>
	    <two_dot_two><![CDATA[inside two_dot_two]]></two_dot_two>
	</two_dot_one>
    </two>
<?someapp just above /root?>
</root>
