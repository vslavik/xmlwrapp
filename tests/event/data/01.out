-- CTOR:
-- START_ELEMENT: root
-- TEXT: 

-- COMMENT:  just under root 
-- TEXT: 
    
-- START_ELEMENT: one
-- TEXT: 
	
-- START_ELEMENT: one_dot_one
-- END_ELEMENT: one_dot_one
-- TEXT: 
    
-- END_ELEMENT: one
-- TEXT: 

    
-- START_ELEMENT: two
-- TEXT: 
	
-- START_ELEMENT: two_dot_one
-- TEXT: 
	    
-- START_ELEMENT: two_dot_two
-- CDATA: inside two_dot_two
-- END_ELEMENT: two_dot_two
-- TEXT: 
	
-- END_ELEMENT: two_dot_one
-- TEXT: 
    
-- END_ELEMENT: two
-- TEXT: 

-- PI: TARGET=someapp DATA=just above /root
-- TEXT: 

-- END_ELEMENT: root
-- DTOR:
